README.md

```diff
@@ -365,60 +365,22 @@ TokenContract.events.Transfer({
 });
 ```
 
-## Deploy
-Starting from version **1.9.4** Parity includes support for running Wasm contracts. Wasm support isn't enabled by default and needs to be specified in the "chainspec" file. `wasmActivationTransition` param sets a block number Wasm support should be activated. This is a sample "development chain" spec with Wasm enabled (based on https://paritytech.github.io/wiki/Private-development-chain):
+## Run node and deploy conract
+Now it's time to deploy our Wasm contract on the blockchain. We can ether test in own [local development chain](#option-1-setup-and-run-development-node) or publish it on the [public Kovan network](#option-2-run-kovan-node).
 
-[Source](https://github.com/paritytech/pwasm-tutorial/tree/master/wasm-dev-chain.json)
-```json
-{
-    "name": "DevelopmentChain",
-    "engine": {
-        "instantSeal": null
-    },
-    "params": {
-        "wasmActivationTransition": "0x01",
-        "gasLimitBoundDivisor": "0x0400",
-        "accountStartNonce": "0x0",
-        "maximumExtraDataSize": "0x20",
-        "minGasLimit": "0x1388",
-        "networkID" : "0x11"
-    },
-    "genesis": {
-        "seal": {
-            "generic": "0x0"
-        },
-        "difficulty": "0x20000",
-        "author": "0x0000000000000000000000000000000000000000",
-        "timestamp": "0x00",
-        "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-        "extraData": "0x",
-        "gasLimit": "0x5B8D80"
-    },
-    "accounts": {
-        "0000000000000000000000000000000000000001": { "balance": "1", "builtin": { "name": "ecrecover", "pricing": { "linear": { "base": 3000, "word": 0 } } } },
-        "0000000000000000000000000000000000000002": { "balance": "1", "builtin": { "name": "sha256", "pricing": { "linear": { "base": 60, "word": 12 } } } },
-        "0000000000000000000000000000000000000003": { "balance": "1", "builtin": { "name": "ripemd160", "pricing": { "linear": { "base": 600, "word": 120 } } } },
-        "0000000000000000000000000000000000000004": { "balance": "1", "builtin": { "name": "identity", "pricing": { "linear": { "base": 15, "word": 3 } } } },
-        "0x004ec07d2329997267ec62b4166639513386f32e": { "balance": "1606938044258990275541962092341162602522202993782792835301376" }
-    }
-}
+### Option 1: Setup and run development node
+Parity **1.9.5** includes support for running Wasm contracts.
+See [instructions](dev-node-setup.md) on how to setup a Wasm-enabled dev node.
 
-```
-Run Parity:
+### Option 2: Run Kovan node
```

Comment:
Maybe just
```
#### Requirements:
- Parity version min. 1.9.5 running fully synced Kovan network  (`parity --chain kovan`)
```

Reply:
and add it to the prerequisites on top ?

Reply:
and add link to the best available description to how to setup parity out there?

Reply:
```
#### Requirements:
- Parity version min. 1.9.5 running fully synced Kovan network  (`parity --chain kovan`)
```
will not conform the narrative of the whole document

> and add link to the best available description to how to setup parity out there?

we have it in Prerequisites